### PR TITLE
VSTS-117 Gulp build script

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

File: tasks/scanner-cli/v3/task.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
```
Comment (henryju) [inline tasks/scanner-cli/v3/task.json]: Looks like you have reformatted all json files. Is it intended? We are loosing all history on files...

Comment (gregaubert) [reply tasks/scanner-cli/v3/task.json]: Ouch, probably happened when I tested the update of the version on v3 tasks files to check if it worked. I'll check if there is an option to not break the current format. And revert those changes.

Comment (henryju) [reply tasks/scanner-cli/v3/task.json]: This is not a big deal if format is changed when packaging the extension

Comment (gregaubert) [reply tasks/scanner-cli/v3/task.json]: Currently it's updating the original file, maybe I misunderstood and we want to update only the packed file ?

Comment (henryju) [reply tasks/scanner-cli/v3/task.json]: In the original script, the task.json files were commited in the repo with version 0.0.0 and never updated. Only the packed file was updated before calling tfx

**Pull request merged**